### PR TITLE
Waiting for react .14 :)

### PR DESCRIPTION
**Pull request by zackify**

### CODE REVIEW

**Comment by JedWatson** [comment]
:tada:

File: tests/testStore.js [hunk 1]
```diff
@@ -1,26 +0,0 @@
-import alt from '../src/alt/alt'
```
Comment (vramana) [inline tests/testStore.js]: @zackify Is this unintended?

Comment (zackify) [inline tests/testStore.js]: these were not supposed to be removed, I did it by accident :(

Comment (vramana) [inline tests/testStore.js]: Okay. Shall add it back ??

Comment (zackify) [inline tests/testStore.js]: that would be great, I can do it if you want

Comment (vramana) [inline tests/testStore.js]: Okay add it then. My PR got some cryptic errors. I'll try to fix them fast.

Comment (zackify) [inline tests/testStore.js]: They may be from the alt stuff, if so I can take a look

Comment (vramana) [inline tests/testStore.js]: No, they got deleted. This is the error. 
```
Warning: ReactDOMComponent: Do not access .props of a DOM node; instead, recreate the
props as `render` did originally or read the DOM properties/attributes directly from this node
(e.g., this.refs.box.className). This DOM node was rendered by `TestComponent`.
```
Trying to find out what is triggering this and the fix?

Comment (vramana) [inline tests/testStore.js]: Didn't read anything regarding this in release notes.

Comment (zackify) [inline tests/testStore.js]: That warning is from React 0.14, we shouldn't be checking the props of a
component, I can fix those tests up once you think the rest is good

Comment (vramana) [inline tests/testStore.js]: All of the failures are related to this I think.

Comment (vramana) [inline tests/testStore.js]: Sorry. Something related to recursive search is broken. I will look into it.

**Pull request merged**